README.md

```diff
@@ -3,6 +3,33 @@
 [![Build Status](https://wso2.org/jenkins/buildStatus/icon?job=products/micro-integrator)](https://wso2.org/jenkins/job/products/job/micro-integrator/)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
+# WSO2 Micro Integrator
+
+## Introduction
+
+In recent times we have witnessed the growth of approaches like Continuous Integration and Continuous Deployment (CI/CD), which has subsequently led to the rise of microservices. Microservices Architecture (MSA) became more popular due to the rise of cloud native technologies and the assistance it provides for agile development. Microservices are designed to start up very quickly, perform a specific task, and then shut down.
+
```

Comment:
"Microservices are designed to start up very quickly, perform a specific task, and then shut down." suites more for serverless technologies which is a subset of the microservices. Therefore we might not be very accurate here.